terraform/account/network.tf

```diff
@@ -94,3 +94,18 @@ resource "aws_route" "private" {
   destination_cidr_block = "0.0.0.0/0"
   nat_gateway_id         = element(aws_nat_gateway.nat.*.id, count.index)
 }
+
+
+resource "aws_db_subnet_group" "data_persistence" {
+  name       = "data-persistence-subnet-${local.account.stack_name}"
```

Comment:
This name-spacing isn't required here as there's only one "stack" (vpc) per account, and only one db_subnet_group.

Reply:
will change based on above proposal.